src/app/location/location.component.ts

```diff
@@ -83,6 +84,12 @@ export class LocationComponent implements OnInit, OnDestroy {
 
           if (data.length) {
             this.loadChartData(0);
+            let lastElement = this.mainData[this.mainData.length - 1];
+            this.totalData = {
+              TotalConfirmed: lastElement.Confirmed,
+              TotalRecovered: lastElement.Recovered,
+              TotalDeaths: lastElement.Deaths,
+            };
```

Comment:
`const { Confirmed, Recovered, Deaths } = lastElement;`

... rest of the logic.

Reply:
if you want to be really fancy:

`const { Confirmed: TotalConfirmed, Recovered: TotalRecovered, Deaths: TotalDeaths } = lastElement;`


and then:

`this.totalData = {
              TotalConfirmed,
              TotalRecovered,
              TotalDeaths
            };`

Reply:
😄

Reply:
Great tip, but just commited changes